ps_linklist.php

```diff
@@ -73,7 +74,7 @@ public function __construct()
     {
         $this->name = 'ps_linklist';
         $this->author = 'PrestaShop';
-        $this->version = '4.0.0';
+        $this->version = '5.0.0';
```

Comment:
Why update it to a major version.
We will have to release a new version soon for Symfony 4.4 compatibility :sweat_smile:

Reply:
Breaking changes, database changes? I don't think it's that big of the issue to move from v5 to v6 in modules? :)